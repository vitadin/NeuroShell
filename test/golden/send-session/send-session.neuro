%% Send command with session management test
%% Tests send functionality with explicit session creation

\session-new test-session
\send Hello from test session
\echo Message history: ${1}
\session-new another-session
\send Hello from another session
\echo Message history: ${1}