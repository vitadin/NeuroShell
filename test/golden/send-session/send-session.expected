Setting _echo_commands = true
%%> \session-new test-session
Created session 'test-session' (ID: 00000001)
%%> \send Hello from test session
This is a mocking reply message for the sending message: Hello from test session
%%> \echo Message history: ${1}
Message history: This is a mocking reply message for the sending message: Hello from test session
%%> \session-new another-session
Created session 'another-session' (ID: 00000004)
%%> \send Hello from another session
This is a mocking reply message for the sending message: Hello from another session
%%> \echo Message history: ${1}
Message history: This is a mocking reply message for the sending message: Hello from another session